frontend/src/v5/ui/controls/sideBar/sidebar.component.tsx

```diff
@@ -35,14 +35,14 @@ export const Sidebar = ({
 	hidden = false,
 	children,
 }: ISidebar): JSX.Element => (
-	<Container className={className} open={open} hidden={hidden}>
+	<SidebarContainer className={className} open={open} hidden={hidden}>
 		{!noButton && (
```

Comment:
I think you can get rid of the `noButton` prop (as well as the conditional render) as it is true when the prop `open` is false, and have just 
```ts
	<SidebarContainer className={className} open={open} hidden={hidden}>
		<ExpandButton onClick={onClick} variant="main">
			<ExpandIcon />
		</ExpandButton>
		<SidebarContent>
			{children}
		</SidebarContent>
	</SidebarContainer>

```

Reply:
I initially did this in case the sidebar is used somewhere else where you want the button to be visible when the sidebar is closed (and act as an open sidebar button). Since it's currently not being used elsewhere I'll get rid of it. If in the future we decide it's needed then it shouldn't be too hard to re-add